cdc/kv/client_test.go

```diff
@@ -2803,11 +2803,11 @@ func (s *etcdSuite) testKVClientForceReconnect(c *check.C) {
 
 	err = failpoint.Enable("github.com/pingcap/ticdc/cdc/kv/kvClientResolveLockInterval", "return(1)")
 	c.Assert(err, check.IsNil)
-	err = failpoint.Enable("github.com/pingcap/ticdc/cdc/kv/kvClientReconnectInterval", "return(3)")
-	c.Assert(err, check.IsNil)
+	originalReconnectInterval := reconnectInterval
+	reconnectInterval = 3 * time.Second
```

Comment:
Is it possible for the race detector to complain about this?

Reply:
No, the test cases run in sequence. (The original code also changes the global value, but triggered by the failpoint)